### DISCUSSION TITLE
Allow exercice type switch when offline

### DISCUSSION THREAD

**Discussion by dindon-sournois** [SethClydesdale/genki-study-resources#134] [Questions] [answered]
Doesn't seem I can change the exercice when offline. I'm not prompted to select the type and I can't change it either during an exercice.
I made sure the option _Skip Exercise Type Selection_ is off.

Is this known? Is this a bug? I would very appreciate this feature as I don't like having to select the correct answer and prefer to write it.

Regards.

**Comment by SethClydesdale** [comment, marked as answer]
Hello @dindon-sournois,

Just to confirm: When doing an exercise, does it say "3rd Edition" under the site title or "lessons-3rd" in the URL?

![3rd](https://user-images.githubusercontent.com/10122549/118809961-4b5ef500-b879-11eb-9df0-bf8dc94ed42a.PNG)

If it does, do let me know which browser this occurs on and I'll take a closer look into it.

If it doesn't, go to the index and click the "3rd Edition" button under page links. This'll take you to the 3rd edition resources which support multiple exercise types.

**Reply by dindon-sournois** [reply]
Argh. You're right I was using the 2nd edition. It works fine with the third. Sorry for bothering you :)

**Reply by SethClydesdale** [reply]
No problem! Happy we got it sorted so can take advantage of the 3rd edition's improvements! 😉
